public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -59,27 +57,27 @@ table(width="100%")
         The context of the binding is implied and is always the
         associated component, so it needs no reference variable.
 
-        For more information see [Template Syntax](../guide/template-syntax.html#interpolation).
+        For more information, see [Template Syntax](../guide/template-syntax.html#interpolation).
```

Comment:
It seems weird to link inside that page without telling you so. I might say something like "the [Interpolation](...) section of the Template Syntax page."